template/build/dev.js

```diff
@@ -0,0 +1,52 @@
+process.env.NODE_ENV = 'development';
+
+const path = require('path');
+const rimraf = require('rimraf');
+const Bundler = require('parcel-bundler');
+const server = require('browser-sync').create();
+
+require('dotenv').config();
+
+const config = require('./studiometa.config');
+
+// Delete dist folder
+rimraf.sync(config.dist);
+
+// Create the dev parcel bundler
+const bundler = new Bundler(config.src, {
+  cacheDir: 'node_modules/.cache',
+  outDir: config.dist,
+  publicUrl: config.public,
+  minify: false,
+  watch: true,
+  hmr: false,
+});
```

Comment:
Does `Bundler` transpile es6 natively ?

Reply:
Yes, it manages natively ES6, Vue and many more (cf. the list of [asset managers](https://github.com/parcel-bundler/parcel/tree/master/packages/core/parcel-bundler/src/assets)).